relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -161,7 +161,7 @@ fn get_transaction_name(
 }
 
 /// These are the tags that are added to all extracted metrics.
-fn extract_universal_tags(
+pub fn extract_universal_tags(
```

Comment:
Exposing this function means that changes to what tags are extracted for metrics by default will also affect profiles. This might be exactly what you want (if the `group by` for profiles should always be the same as for metrics), but it might also be an unwanted side effect, depending on the use case.

Reply:
Right. We were OK with that since getting the normalized transaction name wasn't trivial. Now that we realized we need the original transaction name, reaching to this function to extract tags is unnecessary, we can write our own and not have the dependency.